### PR TITLE
Shopping cart

### PR DESCRIPTION
**Pull request by lvbn**
There will be some conflicts

### CODE REVIEW

File: server/src/routers/users.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
 export default function usersRouter(router: express.Router) {
   router.get("/users", getUsers);
   router.get("/users/:id", getUser);
-  router.post("/user", postUser);
```
Comment (karoteblurs) [inline server/src/routers/users.ts:16]: It should be corrected back to user (single user) without the s

File: server/src/routers/users.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ import {
...
+  router.post("/users", postUser);
```
Comment (karoteblurs) [inline server/src/routers/users.ts]: user

File: server/src/routers/products.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ import {
 
 export default function productsRouter(router: express.Router) {
   router.get("/products", getProducts);
-  router.post("/product", postProduct);
```
Comment (karoteblurs) [inline server/src/routers/products.ts]: same goes with products - only one item can be posted a the time - should be product without the s

File: server/src/routers/products.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ import {
...
+  router.post("/products", postProduct);
```
Comment (karoteblurs) [inline server/src/routers/products.ts]: remove the s

File: server/src/routers/users.ts [hunk 5]
```diff
@@ -12,7 +12,7 @@ import {
...
   router.delete("/users/:id", deleteUser);
```
Comment (karoteblurs) [inline server/src/routers/users.ts:17]: no s

File: server/src/routers/users.ts [hunk 6]
```diff
@@ -12,7 +12,7 @@ import {
...
   router.put("/users/:id", updateUser);
```
Comment (karoteblurs) [inline server/src/routers/users.ts:18]: no s

**Review by karoteblurs — CHANGES_REQUESTED** [review]

File: server/src/models/models.ts
Comment (jamalbran) [inline server/src/models/models.ts:1]: There will be a conflict with dev branch.
The last update in dev include the import of both transaction models, but it's something very easy to solve at the merge

**Review by jamalbran — APPROVED** [review]

File: server/src/models/models.ts
Comment (lvbn) [reply server/src/models/models.ts:1]: Alright, we'll handle that

File: server/src/routers/users.ts (hunk 1 above)
Comment (lvbn) [reply server/src/routers/users.ts:16]: it's corrected now. Thanks for your collaboration

File: server/src/routers/users.ts (hunk 2 above)
Comment (lvbn) [reply server/src/routers/users.ts]: corrected : )

File: server/src/routers/products.ts (hunk 3 above)
Comment (lvbn) [reply server/src/routers/products.ts]: also done : )

File: server/src/routers/products.ts (hunk 4 above)
Comment (lvbn) [reply server/src/routers/products.ts]: Ok, thanks

File: server/src/routers/users.ts (hunk 5 above)
Comment (lvbn) [reply server/src/routers/users.ts:17]: Alright, thanks for pointing it out. Already fixed

File: server/src/routers/users.ts (hunk 6 above)
Comment (lvbn) [reply server/src/routers/users.ts:18]: Ok thank you, it's fixed now : )

**Review by sherrilla71940 — APPROVED** [review]

**Review by jamalbran — APPROVED** [review]

**Pull request merged**